packages/javascript-sdk/src/config/index.ts

```diff
@@ -79,6 +79,7 @@ abstract class Config {
         'Missing well-known property. Use `Config.set` method if not using well-known endpoint.',
       );
     }
+    // @ts-expect-error safety against runtimes without typescript
```

Comment:
I left this here because while typescript says we can not have this happen, i figured it was safer for us to keep this check in the code for runtime and using expect error just means we expect this line to error (because we are checking a property that can't exist on the interface)